### PR TITLE
added sort operation

### PR DESCRIPTION
**Pull request by Vijeth-eg**
Created methods for sorting operation(ascending and descending)

### CODE REVIEW

File: GitHubMainApplication/sortOperation.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            else
+            {
+                Console.WriteLine("array is empty");
+            }
+
+        }
+        int temperoryVariable,count=1;
+
+        public  void SortAscending(int[] array) {
+            for (int i = 0; i < array.Length; i++)
```
Comment (tshet-eg) [inline GitHubMainApplication/sortOperation.cs:35]: apply appropriate naming conventions

File: GitHubMainApplication/sortOperation.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GitHubMainApplication
+{
+    public class SortOperation
+    {
+        public SortOperation(int[] array) {
+            if (array.Length != 0)
```
Comment (tshet-eg) [inline GitHubMainApplication/sortOperation.cs:12]: change the if condition to count!=0, 
also add another parameter to the function that reads the count from the main class

**Review by tshet-eg — COMMENTED** [review]
Please make necessary changes

File: GitHubMainApplication/sortOperation.cs (hunk 2 above)
Comment (Vijeth-eg) [reply GitHubMainApplication/sortOperation.cs:12]: ill update it with count variable once main method is compleated

File: GitHubMainApplication/sortOperation.cs (hunk 1 above)
Comment (Vijeth-eg) [reply GitHubMainApplication/sortOperation.cs:35]: have appropriate naming convention except i and j in for loop

**Pull request merged**